sympy/series/sequences.py

```diff
@@ -424,7 +424,8 @@ class SeqExpr(SeqBase):
 
     >>> from sympy.series.sequences import SeqExpr
     >>> from sympy.abc import x
-    >>> s = SeqExpr((1, 2, 3), (x, 0, 10))
+    >>> from sympy.core.containers import Tuple
+    >>> s = SeqExpr(Tuple(1, 2, 3), Tuple(x, 0, 10))
```

Comment:
Maybe this class should sympify its args

Reply:
It actually doesn't have a `__new__` method at all and the docstring says `Various sequences should inherit from this class.` So I interpreted it as a class that should normally not be used by users (similar to `Basic`).

Reply:
Do the subclasses sympify their arguments?

Reply:
Yes, there were also no tests that had to be changed for the subclasses as far as I can remember. They do use `sympify` over `_sympify` though (and are fairly complicated).